tests/ConverterTest.php

```diff
@@ -14,7 +14,7 @@
 if (is_dir(CONV_ROOT.'fluxbb/'))
 	define('PUN_ROOT', CONV_ROOT.'fluxbb/');
 
-require_once 'PHPUnit/Framework/TestCase.php';
+require_once 'PHPUnit/Autoload.php';
```

Comment:
The build was broken even without this change. I thought this change makes pass the test.